docs/ref/contrib/auth.txt

```diff
@@ -279,11 +284,15 @@ Manager methods
 
         See :ref:`Creating users <topics-auth-creating-users>` for example usage.
 
-    .. method:: create_superuser(username, email, password, **extra_fields)
+    .. method:: create_superuser(username, email=None, password=None, **extra_fields)
```

Comment:
I guess here was could add a `As with `create_user()...`

Reply:
Wouldn't a repeat of the `admonition` be better, or are these to be used sparingly?